pricing-page-exercise/saffet/styles.css

```diff
@@ -0,0 +1,257 @@
+body {
+    font-family: sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+/* Navigation bar*/
+.nav-bar{
+    box-sizing: border-box;
+    height: 70px;
+    border: 1px transparent grey;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-left: 25px;
+    padding-right: 35px;
+    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+}
+
+.nav-bar > p{
+    font-size: 19px; /*company name*/
+}
+
+.nav-links{
+    display: flex;
+    align-items: center;
+}
+
+.nav-links a{
+    text-decoration: none;
+}
+
+.nav-links>a{
+    padding-left: 25px;
+    color: black;
+}
+
+.nav-links>a:hover{
+    text-decoration: underline;
+}
+
+.button{
+    margin-left: 25px;
+    border: 1px solid #007bff;
+    padding: 10px 14px;
+    border-radius: 4px;
+}
+
+.button a{
+    color: #007bff;
+}
+
+.button:hover{
+    background-color: #007bff;
+}
+
+.button:hover a{
+    color: white;
+}
+
+/* Main Content includes Header, text and cards */
+.description{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.header{
+    font-size: 56px;
+    font-weight: 300;
+    margin-top: 70px;
+    margin-bottom: 10px;
+}
+
+.desc-text{
+    font-size: 1.2rem;
+    font-weight: 200;
+    max-width: 45%;
+    text-align: center;
+    margin-top: 0;
+    line-height: 30px;
+}
+main{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+/* 
+.cards{
```

Comment:
It's better not to have commented out code (it's ok to have it in your own project for learning purposes but you'll mostly probably won't be allowed to leave it at work if you company has decent code standards).